app/views/users/view.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Users#view</h1>
+<p>Find me in app/views/users/view.html.erb</p>
```

Comment:
- [ ] Please delete this view because you only asked to create index and show views for users.